node/pkg/solana/client.go

```diff
@@ -534,6 +536,16 @@ func (s *SolanaWatcher) processMessageAccount(logger *zap.Logger, data []byte, a
 	var txHash eth_common.Hash
 	copy(txHash[:], acc[:])
 
+	var reliable bool
+	switch string(data[:3]) {
+	case accountPrefixReliable:
+		reliable = true
+	case accountPrefixUnreliable:
+		reliable = false
+	default:
+		panic("invalid prefix")
```

Comment:
Do we really want to panic here, rather than just log an error and assume reliable? Are there upgrade scenarios where some guardians might see a new message type that they don't support yet?

Reply:
we can safely panic here as we only reach this code after having checked this right now